packages/backend/src/peripherals/database/ForcedTransactionsRepository.ts

```diff
@@ -365,15 +425,18 @@ export class ForcedTransactionsRepository extends BaseRepository {
         .update({
           finalize_hash: finalizeHash.toString(),
         })
-      await trx('transaction_status').insert({
-        hash: finalizeHash.toString(),
-        mined_at:
-          minedAt !== null && minedAt !== undefined
-            ? BigInt(minedAt.toString())
-            : null,
-        sent_at: sentAt !== null ? BigInt(sentAt.toString()) : null,
-        block_number: blockNumber,
-      })
+      await trx('transaction_status')
+        .insert({
+          hash: finalizeHash.toString(),
+          mined_at:
+            minedAt !== null && minedAt !== undefined
+              ? BigInt(minedAt.toString())
+              : null,
+          sent_at: sentAt !== null ? BigInt(sentAt.toString()) : null,
+          block_number: blockNumber,
+        })
+        .onConflict()
+        .ignore() // TODO: not sure
```

Comment:
What's up with this?

Reply:
We prefer to ignore if conflicted on primary key, because there's a possibility we could override additional information (sent_at).